.prettierrc

```diff
@@ -4,5 +4,6 @@
   "semi": true,
   "singleQuote": true,
   "trailingComma": "es5",
-  "arrowParens": "avoid"
+  "arrowParens": "avoid",
+  "endOfLine": "auto"
```

Comment:
please test this solution in `add-link-component` branch,
go to `src/Link/Link.tsx` file and run these scripts:
- `yarn lint --fix`
- `yarn format`
you will see both scripts formats the file with different formats so issue still exist.